core/src/main/resources/i18n/displayStrings.properties

```diff
@@ -903,6 +905,56 @@ portfolio.pending.mediationResult.popup.selfAccepted.lockTimeOver=You have accep
+  send the fiat or altcoin payment to the BTC seller. Open a mediation ticket by clicking cmd+o (or ctrl+o). \
+  The mediator should make a suggested payout so that the seller receives their trade amount back and both traders get \
+  back their security deposit. That way there is no security risk and only the trade fee is lost. \
+  You can request at https://github.com/bisq-network/support/issues for getting reimbursed the lost trade fee.
+portfolio.pending.failedTrade.seller.existingDepositTxButMissingDelayedPayoutTx=The delayed payout transaction is missing.\n\n\
+  Without that arbitration cannot be opened. The funds have been locked up in the deposit transaction. If the buyer is \
+  also missing the delayed payout transaction he is instructed to not send the payment but open a mediation ticket. \
+  You should open a mediation ticket as well by clicking cmd+o (or ctrl+o). \
+  In case that the buyer has not sent the payment the mediator should make a suggested payout so that you receive \
+  your trade amount back and both traders get back their security deposit. Otherwise the trade amount should go to the buyer. \
+  You can request at https://github.com/bisq-network/support/issues for getting reimbursed the lost trade fee.
+portfolio.pending.failedTrade.errorMsgSet=There have been an error during the trade protocol execution.\n\n\
+  Error: {0}\n\n\
+  This error might not be critical and the trade could be completed normally. If you are unsure you can open a mediation \
+  ticket to get advice from the Bisq mediators. If the error was critical and the trade cannot be completed you might \
+  have lost your trade fee. If so, you can request at https://github.com/bisq-network/support/issues for \
+  getting reimbursed the lost trade fee.
+portfolio.pending.failedTrade.missingContract=The trade contract is not set.\n\n\
+  The trade cannot be completed and you might \
+  have lost your trade fee. If so, you can request at https://github.com/bisq-network/support/issues for \
+  getting reimbursed the lost trade fee.
+portfolio.pending.failedTrade.info.popup=The trade protocol encountered some problems.\n\n{0}
+portfolio.pending.failedTrade.txChainInvalid.moveToFailed=The trade protocol encountered a serious problem.\n\n{0}\n\n\
+  Do you want to move the trade to failed trades?\n\n\
+  You cannot open mediation or arbitration from the failed trades view, but you can move it back to the open \
+  trades screen any time.
+portfolio.pending.failedTrade.txChainValid.moveToFailed=The trade protocol encountered some problems.\n\n{0}\n\n\
+  The trade transactions have been published and funds are locked up. Only move the trade to failed trades if you are \
+  really sure. It might prevent options to resolve the problem.\n\n\
+  Do you want to move the trade to failed trades?\n\n\
+  You cannot open mediation or arbitration from the failed trades view, but you can move it back to the open \
```

Comment:
```suggestion
  The trade transactions have been published and funds are locked. Only move the trade to failed trades if you are \
  really sure. It might prevent options to resolve the problem.\n\n\
  Do you want to move the trade to failed trades?\n\n\
  You cannot open mediation or arbitration from the failed trades view, but you can move a failed trade back to the open \
```